internal/pkg/agent/application/upgrade/artifact/download/http/downloader.go

```diff
@@ -208,8 +206,9 @@ func (e *Downloader) downloadFile(ctx context.Context, artifactName, filename, f
 		}
 	}
 
+	lpObs := newLoggingProgressObserver(e.log, e.config.HTTPTransportSettings.Timeout)
```

Comment:
wild name, i had to go up here to figure it what it means

Reply:
Fixed in e9fc0970bac5927bd135c4f0a78b36b74d1eae5a.